tests/quality_pragmaitc/test_quality_constraint.py

```diff
@@ -1,35 +1,40 @@
-from goald.quality.pragmatic.model.qualityConstraint import QualityConstraint
-from goald.quality.pragmatic.model.metric import CommonMetrics
+from goald.quality.pragmatic.model.quality_constraint import QualityConstraint
+from goald.quality.pragmatic.model.common_metrics import CommonMetrics
 from goald.quality.pragmatic.model.context import Context
+from goald.quality.pragmatic.model.comparison import Comparison
+
 
 def test_quality_constraint():
-    commonMetrics = CommonMetrics()
+    qc = QualityConstraint(
+        Context("C1"), CommonMetrics.SECONDS, 15, Comparison.LESS_THAN)
+    assert True is qc.abidesByQC(13, CommonMetrics.SECONDS)
+    assert False is qc.abidesByQC(16, CommonMetrics.SECONDS)
 
-    qc =  QualityConstraint(Context("C1"), commonMetrics.SECONDS, 15, 'LESS_THAN')
-    assert True == qc.abidesByQC(13, commonMetrics.SECONDS)
-    assert False == qc.abidesByQC(16, commonMetrics.SECONDS)
 
 def test_should_select_stricter_constraint():
-    commonMetrics = CommonMetrics()
+    lessStrictQC = QualityConstraint(
+        Context("C1"), CommonMetrics.SECONDS, 15, Comparison.LESS_THAN)
+    moreStrictQC = QualityConstraint(
+        Context("C2"), CommonMetrics.SECONDS, 10, Comparison.LESS_THAN)
 
-    lessStrictQC = QualityConstraint(Context("C1"), commonMetrics.SECONDS, 15, 'LESS_THAN')
-    moreStrictQC = QualityConstraint(Context("C2"), commonMetrics.SECONDS, 10, 'LESS_THAN')
+    assert moreStrictQC is lessStrictQC.stricterQC(moreStrictQC)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)